tmt/steps/execute/internal.py

```diff
@@ -279,7 +281,7 @@ def _run_tests(self, guest, extra_environment=None):
                 source=self.data_path(test, full=True),
                 extend_options=exclude)
 
-            result = self.check(test)
+            results = self.check(test) # Produce list of results
             duration = click.style(test.real_duration, fg='cyan')
```

Comment:
This is current output. Should we omit duration for subtests? Repeating same value isn't right.
We know 'real_duration' only if there is single result.

```
execute
        how: tmt
        exit-first: False
            test: Test provides custom results
                00:00:00 pass /test/custom-results/test/one [1/1]
                00:00:00 fail /test/custom-results/test/two [1/1]
        summary: 2 tests executed
```

Reply:
Done, duration provided in results.yaml is reporter here. If no duration is provided, 8 spaces is there to keep an indentation.